### ISSUE TITLE
[Fieldgroup Request] table 5955 "Skill Code"

### ISSUE THREAD

**Issue by guidorobben** [microsoft/ALAppExtensions#18721]
Hi,
please add a dropdown fieldgroup to the table 5955 "Skill Code". We want to extend the DropDown fieldgroup, but before we can do this, there has to be one. 

fieldgroups
{
fieldgroup(DropDown; Code, Description)
{}
}

**Comment by JesperSchulz** [comment]
Thanks for reporting this. We agree, and we’ll publish a fix asap, either in an update for the current version or in the next major release. Please do not reply to this, as we do not monitor closed issues. If you have follow-up questions or requests, please create a new issue where you reference this one.

Build ID: 42571.
